libvast/src/system/counter.cpp

```diff
@@ -23,9 +23,11 @@ counter_state::counter_state(caf::event_based_actor* self) : super(self) {
 
 void counter_state::init(expression expr, index_actor index,
                          bool skip_candidate_check) {
-  auto q = vast::query::make_count(
-    self_, skip_candidate_check ? query::count::estimate : query::count::exact,
-    std::move(expr));
+  auto q = vast::query_context::make_count(self_,
```

Comment:
yes, since variable names can not be found via refactoring easily, I was relying on text search for `query` in some cases. There may be more of these, I'll try to find some more.

Reply:
I'm impressed, `clangd` finds those spots via `references`, wasn't expecting this.